src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -110,7 +112,8 @@
 
     private List<PodImagePullSecret> imagePullSecrets = new ArrayList<PodImagePullSecret>();
 
-    private transient List<ToolLocationNodeProperty> nodeProperties;
+    @SuppressFBWarnings(value = "SE_BAD_FIELD")
```

Comment:
is this actually needed ? doesn't fail without it

Reply:
I indeed get a FindBugs. However ignoring it feels like hack.

In Jenkins core, this is handled using a `DescribableList` (see [here](https://github.com/jenkinsci/jenkins/blob/0268cc99293ba985fe0a919e8c0633e33174f049/core/src/main/java/jenkins/model/Jenkins.java#L688)). I would recommend sticking to the same pattern.